ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html

```diff
@@ -18,9 +18,9 @@
 
 <div *ngIf="isOpen()" [@slideUpDown] [@.disabled]="creationMode">
   <mat-card class="save-bar__content" [class.mat-elevation-z3]="creationMode" [class.mat-elevation-z6]="!creationMode">
-    <div class="save-bar__content__label">{{ creationMode ? '' : 'Please note you have unsaved modifications.' }}</div>
+    <div class="save-bar__content__label">{{ creationMode ? '' : 'You have unsaved changes.' }}</div>
```

Comment:
Label text: You have changes that are not saved (without the ".")

Reply:
Seen with Lju we will go with `You have unsaved changes`